.babelrc

```diff
@@ -1,19 +1,25 @@
 {
-  "plugins": [
-    ["transform-runtime", {
-      "polyfill": false,
-      "regenerator": true
-    }]
+  "presets": [
+    ["@babel/env", {
+      "loose": true,
+      "useBuiltIns": "usage",
+      "targets": {
+        "browsers": ["last 2 versions", "safari >= 7", "ie 9"],
+        "node": "4"
+      },
+      "exclude": [
+        "es6.map", "es6.set", "es6.function.name", "es6.symbol",
+        "es6.regexp.split", "es6.regexp.replace", "es6.regexp.match",
+        "es6.string.repeat", "es6.object.set-prototype-of", "es6.regexp.flags"
```

Comment:
I wonder what happens here. It seems like `.babelrc` config becomes more complicated than it should...

Reply:
Babel is a bit over-agressive in including builtins, which balloons the size of the bundles. There are trade-offs in life: sometimes you get a more complicated babelrc to get a smaller js bundle.

Reply:
I'd just say that excluding features this way is slightly dangerous. Using presets is a more accurate approach. But I guess since it is tested on older Node instances, it shouldn't be an issue...